FlipperZero/FlipperZero.xcodeproj/project.pbxproj

```diff
@@ -180,6 +198,8 @@
 			children = (
 				F0DBFA1A24EF2F9600EB2880 /* FlipperZeroApp.swift */,
 				F0DBFA1C24EF2F9900EB2880 /* Assets.xcassets */,
+				49CE8F9D25262E2300B9CBE4 /* LaunchScreen.storyboard */,
```

Comment:
This is iOS-only storyboard and it breaks macOS compilation. Please remove it from the macOS target, and move the file to the `iOS` directory